### ISSUE TITLE
Crash because of wrong signature of DefineBlenderMesh

### ISSUE THREAD

**Issue by omgold** [LuxCoreRender/BlendLuxCore#306]
#### System Information

Arch Linux

#### Software Version

Blender: 2.80
LuxCore: 2.2 (with patches from AUR package, needed to build with Arch version of Boost and Python)
BlendLuxCore: 2.2
Python: 3.7.4
Compiler: gcc 9.1.0
Boost: 1.69.0

#### Error message from Blender console

```
BlendLuxCore registered (2.2)

==================================================
[Engine/Final] Rendering layer "RenderLayer"
[Exporter] Creating session
[SDL][53.252] Camera type: perspective
[SDL][53.252] Camera position: Point[2.21022, -3.86524, 0.725669]
[SDL][53.252] Camera target: Point[1.61942, -3.06329, 0.637213]
[SDL][53.252] Camera clipping plane disabled
Traceback (most recent call last):
  File "/usr/share/blender/2.80/scripts/addons/luxcorerender/engine/base.py", line 84, in render_final
    final.render(self, depsgraph)
  File "/usr/share/blender/2.80/scripts/addons/luxcorerender/engine/final.py", line 49, in render
    _render_layer(engine, depsgraph, statistics, layer)
  File "/usr/share/blender/2.80/scripts/addons/luxcorerender/engine/final.py", line 61, in _render_layer
    engine.session = engine.exporter.create_session(depsgraph, engine=engine, view_layer=view_layer)
  File "/usr/share/blender/2.80/scripts/addons/luxcorerender/export/__init__.py", line 112, in create_session
    if not self.object_cache2.first_run(self, depsgraph, view_layer, engine, luxcore_scene, scene_props, is_viewport_render):
  File "/usr/share/blender/2.80/scripts/addons/luxcorerender/export/caches/object_cache.py", line 60, in first_run
    luxcore_scene, scene_props, is_viewport_render)
  File "/usr/share/blender/2.80/scripts/addons/luxcorerender/export/caches/object_cache.py", line 105, in _convert_obj
    luxcore_scene, scene_props, is_viewport_render)
  File "/usr/share/blender/2.80/scripts/addons/luxcorerender/export/caches/object_cache.py", line 136, in _convert_mesh_obj
    is_viewport_render, use_instancing, transform)
  File "/usr/share/blender/2.80/scripts/addons/luxcorerender/export/mesh_converter.py", line 40, in convert
    material_count, mesh_transform)
Boost.Python.ArgumentError: Python argument types in
    Scene.DefineBlenderMesh(Scene, str, int, int, int, int, int, int, int, int, list)
did not match C++ signature:
    DefineBlenderMesh(luxcore::detail::SceneImpl*, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, unsigned long, unsigned long, unsigned long, unsigned long, unsigned long, unsigned long)
    DefineBlenderMesh(luxcore::detail::SceneImpl*, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, unsigned long, unsigned long, unsigned long, unsigned long, unsigned long, unsigned long, boost::python::api::object)
ERROR: Python argument types in
    Scene.DefineBlenderMesh(Scene, str, int, int, int, int, int, int, int, int, list)
did not match C++ signature:
    DefineBlenderMesh(luxcore::detail::SceneImpl*, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, unsigned long, unsigned long, unsigned long, unsigned long, unsigned long, unsigned long)
    DefineBlenderMesh(luxcore::detail::SceneImpl*, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, unsigned long, unsigned long, unsigned long, unsigned long, unsigned long, unsigned long, boost::python::api::object)
```

#### Short description of error

Attempt to render crashes plugin

#### Exact steps for others to reproduce the error

- open default blender startup file
- switch renderer to LuxCore (Device: CPU, Engine: Path)
- do Render Image / F12

**Comment by Dade916** [comment]
Have tried to use the official v2.2 binaries in the same setup ? Does it happen with them too ?

You should use our static building environment https://github.com/LuxCoreRender/LinuxCompile. We don't support building LuxCore with not tested external libraries (because, over the years, we have run over all kind of problems related to bugs and incompatibilities in newer/older external libraries).

**Comment by omgold** [comment]
I found the official binary to work. Rebuilding with the build environment turned out to be very difficult, due to version mismatches with system libraries, so I finally gave up.

**Comment by bartoszek** [comment]
Fixed in #307
